Description: Start mutter as early as possible, since with plugins (such as gnome-shell) it will take the longest time to start up; Let the scheduler deal with squeezing the other processes into mutter's idle periods.
Author: Martin Pitt <martin.pitt@ubuntu.com>

Index: mutter-2.28.1~git20091208/src/mutter-wm.desktop
===================================================================
--- mutter-2.28.1~git20091208.orig/src/mutter-wm.desktop	2010-01-26 16:17:07.000000000 +0100
+++ mutter-2.28.1~git20091208/src/mutter-wm.desktop	2010-01-26 16:17:39.000000000 +0100
@@ -22,7 +22,7 @@
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=mutter
 X-GNOME-Bugzilla-Component=general
-X-GNOME-Autostart-Phase=WindowManager
+X-GNOME-Autostart-Phase=Initialization
 X-GNOME-Provides=windowmanager
 X-GNOME-Autostart-Notify=true
 
Index: mutter-2.28.1~git20091208/src/mutter-wm.desktop.in
===================================================================
--- mutter-2.28.1~git20091208.orig/src/mutter-wm.desktop.in	2010-01-26 16:17:02.000000000 +0100
+++ mutter-2.28.1~git20091208/src/mutter-wm.desktop.in	2010-01-26 16:17:32.000000000 +0100
@@ -11,7 +11,7 @@
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=mutter
 X-GNOME-Bugzilla-Component=general
-X-GNOME-Autostart-Phase=WindowManager
+X-GNOME-Autostart-Phase=Initialization
 X-GNOME-Provides=windowmanager
 X-GNOME-Autostart-Notify=true
 
Index: mutter-2.28.1~git20091208/src/mutter.desktop
===================================================================
--- mutter-2.28.1~git20091208.orig/src/mutter.desktop	2010-01-26 16:17:10.000000000 +0100
+++ mutter-2.28.1~git20091208/src/mutter.desktop	2010-01-26 16:17:26.000000000 +0100
@@ -23,6 +23,6 @@
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=mutter
 X-GNOME-Bugzilla-Component=general
-X-GNOME-Autostart-Phase=WindowManager
+X-GNOME-Autostart-Phase=Initialization
 X-GNOME-Provides=windowmanager
 X-GNOME-Autostart-Notify=true
Index: mutter-2.28.1~git20091208/src/mutter.desktop.in
===================================================================
--- mutter-2.28.1~git20091208.orig/src/mutter.desktop.in	2010-01-26 16:17:05.000000000 +0100
+++ mutter-2.28.1~git20091208/src/mutter.desktop.in	2010-01-26 16:17:18.000000000 +0100
@@ -12,6 +12,6 @@
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=mutter
 X-GNOME-Bugzilla-Component=general
-X-GNOME-Autostart-Phase=WindowManager
+X-GNOME-Autostart-Phase=Initialization
 X-GNOME-Provides=windowmanager
 X-GNOME-Autostart-Notify=true
